src/View.zig

```diff
@@ -0,0 +1,161 @@
+//! A View is effectively an "oversized" Window that can be written to and rendered in pieces.
+
+const std = @import("std");
+const mem = std.mem;
+
+const View = @This();
+
+const Screen = @import("Screen.zig");
+const Window = @import("Window.zig");
+const Unicode = @import("Unicode.zig");
+const Cell = @import("Cell.zig");
+
+/// View Allocator
+alloc: mem.Allocator,
```

Comment:
I don't think we need the allocator either. This isn't used except for `deinit`, so let's just force callers to pass in their allocator for deinit.

Reply:
The one other place it's used is to store the Screen as a whole. Which allows Views to be a little more locked down after initialization. 

I'm also a bigger fan of Managed structs in this instance as they prevent users from accidentally passing the wrong Allocator back in for deallocation later. But that's largely subjective, and not a hill I'm willing to die on, haha